helm/fiftyone-teams-app/templates/api-deployment.yaml

```diff
@@ -1,3 +1,7 @@
+{{- $serviceAccountName := include "fiftyone-teams-app.serviceAccountName" . }}
+{{- if .Values.apiSettings.rbac.create }}
+{{- $serviceAccountName = include "rbac-do-templates.service-account-name" . }}
```

Comment:
What if `create` is false, but a name is configured in values? I believe that should indicate that the user wants to configure an existing SA, which should be used but just not created.